src/db/seeds.sql

```diff
@@ -0,0 +1,22 @@
+DROP DATABASE IF EXISTS node_db;
+
+CREATE DATABASE node_db;
+
+\connect node_db
+
+DROP TABLE IF EXISTS "Users";
+
+CREATE TABLE "Users" (
+  id SERIAL NOT NULL,
+  login VARCHAR(128) NOT NULL,
+  password VARCHAR(128) NOT NULL,
+  age INTEGER NOT NULL,
+  "isDeleted" BOOLEAN NOT NULL DEFAULT false,
+  PRIMARY KEY (id)
+);
+
+INSERT INTO "Users" (login, password, age) values ('anna123@gmail.com', 'fhgU3fdg', 12);
```

Comment:
It is better to separate in different files SQL for database structure initialization and SQL for test data upload

Reply:
Done